src/AcceptanceTests/TestSuiteConstraints.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using AcceptanceTesting.Support;
+
+    public partial class TestSuiteConstraints
+    {
+        public bool SupportsDtc { get; } = false;
+        public bool SupportsCrossQueueTransactions { get; } = true; // TODO: verify that this is an AtomicSendWithReceive. If not, set to false
```

Comment:
Follow up, this test is green because core does not invoke the dispatcher for the send right before the exception. We are not effectively sending via (we are logging that we do though, I'll fix the log separatly)